### PR TITLE
Add sys tables

### PR DESCRIPTION
**Pull request by zihaoAK47**
Refer #26106.

Changes proposed in this pull request:
  - Add Mysql host_summary_by_statement_type table
  - Add sys_config and host_summary_by_statement_type test

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ✓] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ✓] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ✓] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ✓] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by zihaoAK47** [comment]
@strongduanmu Hi mentor, I tried to submit a pull request for sys.host_summary_by_statement_type. please review

**Comment by strongduanmu** [comment]
Hi @zihaoAK47, can you comment on the issue? I will assign it to you.

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public enum SystemSchemaBuilderRule {
             "socket_summary_by_event_name", "socket_summary_by_instance", "status_by_account", "status_by_host", "status_by_thread", "status_by_user", "table_handles",
             "table_io_waits_summary_by_index_usage", "table_io_waits_summary_by_table", "table_lock_waits_summary_by_table", "threads", "user_variables_by_thread", "users", "variables_by_thread"))),
     
-    MYSQL_SYS("MySQL", "sys", new HashSet<>(Collections.singleton("sys"))),
+    MYSQL_SYS("MySQL", "sys", new HashSet<>(new HashSet<>(Arrays.asList("sys", "host_summary_by_statement_type")))),
```
Comment (strongduanmu) [inline infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java:73]: Why it has two hashset?

File: test/e2e/sql/src/test/resources/cases/dql/dataset/db/select_mysql_sys_host_summary_by_statement_type.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <column name="total" />
+        <column name="total_latency" />
+        <column name="max_latency" />
+        <column name="lock_latency" />
+        <column name="rows_sent" />
+        <column name="rows_examined" />
+        <column name="rows_affected" />
+        <column name="full_scans" />
+    </metadata>
+</dataset>
```
Comment (strongduanmu) [inline test/e2e/sql/src/test/resources/cases/dql/dataset/db/select_mysql_sys_host_summary_by_statement_type.xml:31]: Please add new line at the end of file.

File: test/e2e/sql/src/test/resources/cases/dql/dataset/db/select_mysql_sys_sys_config.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  -->
+
+<dataset>
+    <metadata>
+        <column name="variable" />
+        <column name="value" />
+        <column name="set_time" />
+        <column name="set_by" />
+    </metadata>
+</dataset>
```
Comment (strongduanmu) [inline test/e2e/sql/src/test/resources/cases/dql/dataset/db/select_mysql_sys_sys_config.xml:25]: Please add new line at the end of file.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java (hunk 1 above)
Comment (zihaoAK47) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java]: Sorry for being careless, it has been corrected :astonished:

**Comment by boyjoy1127** [comment]
Will this issue be completed before June 15 which is the final date of version 5.4.0?

**Pull request closed without merging**